src/applications/simple-forms/20-10206/tests/unit/config/form.unit.spec.js

```diff
@@ -0,0 +1,45 @@
+import { expect } from 'chai';
+import formConfig from '../../../config/form';
+
+describe('formConfig', () => {
+  let isLocalhostStub;
+
+  beforeEach(() => {
+    delete require.cache[require.resolve('../../../config/form')];
+  });
+
+  afterEach(() => {
+    if (isLocalhostStub) {
```

Comment:
Wrong!  We do need this guard.  Otherwise, one of the tests throws an error: "after each" hook for "should have a rootUrl property": TypeError: Cannot read property 'restore' of undefined